README.md

```diff
@@ -42,6 +42,34 @@ In the installation screen, select a minimum of:
 
 All other components are optional.
 
+#### Local DB
+
+Find out if Local DB is already installed
+
+```Batchfile
+powershell -NoProfile sqllocaldb -?
+```
+
+If the command fails or is not existant then Local DB is not installed or needs to be reconfigured. Follow the steps below:
+
+* Download SQL Express [installer](https://www.microsoft.com/en-us/sql-server/sql-server-editions-express)
+* Select the Download Media option
+* Select LocalDB from the option `WHICH PACKAGE WOULD YOU LIKE TO DOWNLOAD`
+* Install `SqlLocalDB.msi` from the download location
```

Comment:
I don't see a "Download Media" option or the text `WHICH PACKAGE WOULD YOU LIKE TO DOWNLOAD`.

This may be going into too much detail, and the web page may change around in the future. E.g. for VS, we just say "Install Visual Studio" and hyperlink it. How about replacing these bullets with:

> Install [SQL Server Express 2017 LocalDB](https://www.microsoft.com/en-us/sql-server/sql-server-editions-express). Note that the default download is probably the full version of SQL Server Express, which is not required. You may have to scroll down to the find the "LocalDB" version.